Fixed fr.yaml + sampleUtterances.fr

MehdyDriouech opened the pull request.

fergyfresh commented:
I love that you're trying so hard to give us French support, but you're missing the point of a few of the things we need to happen in order to make this work. I have a hard time believing many of the features work if you've tried to test it out.

There are two main portions of how we have configuration files that can control the voice input and the voice output to the app.

The voice input is controlled by the intents and their corresponding `speech_assets/sampleUtterances.fr.txt` this is where you will be able to change the words and languages you can speak into your Alexa and have it understand you. It is very important that you do not change the first 'word' on every line in this file. For example you changed some occurrences of `GeeMusicExampleForComment` to `GeeMusiclisExampleForComment` in your sample utterances file. The two things that should never change is any word GeeMusicXXX in it and anything inside of curly braces in this file. You can change the ordering of the items in the  curly braces `{ }`, but you need them for it to work.

If you just want to speak French to it and are ok with the alexa speaking English back to you (I'm not actually sure that it can yet.) you can just mimic what a Japanese speaking contributor did on this PR that was merged here https://github.com/stevenleeg/geemusic/commit/05425dbe452620d654a055311e60d5e367bf5049.

The second part is the `fr.yaml` file which should be an EXACT copy of the `en.yaml` with the things in quotes translated to French. In this PR you changed the formatting of the file, changed the order of some template strings, and changed some template strings. You also changed some of the variables too (Items in double curly braces). Take a note from the merges that were successfully incorporated and try to make this one match. Also please test and make sure that the `fr.yaml` actually works as I don't speak French and I wouldn't be able to really test it.

MehdyDriouech commented:
i will do it more precisely, sorry for the errors.

Le mer. 13 juin 2018 à 21:09, Billy Ferguson <notifications@github.com> a
écrit :

> I love that you're trying so hard to give us French support, but you're
> missing the point of a few of the things we need to happen in order to make
> this work. I have a hard time believing many of the features work if you've
> tried to test it out.
>
> There are two main portions of how we have configuration files that can
> control the voice input and the voice output to the app.
>
> The voice input is controlled by the intents and their corresponding
> speech_assets/sampleUtterances.fr.txt this is where you will be able to
> change the words and languages you can speak into your Alexa and have it
> understand you. It is very important that you do not change the first
> 'word' on every line in this file. For example you changed some occurrences
> of GeeMusicExampleForComment to GeeMusiclisExampleForComment in your
> sample utterances file. The two things that should never change is any word
> GeeMusicXXX in it and anything inside of curly braces in this file. You can
> change the ordering of the items in the curly braces { }, but you need
> them for it to work.
>
> If you just want to speak French to it and are ok with the alexa speaking
> English back to you (I'm not actually sure that it can yet.) you can just
> mimic what a Japanese speaking contributor did on this PR that was merged
> here 05425db
> <https://github.com/stevenleeg/geemusic/commit/05425dbe452620d654a055311e60d5e367bf5049>
> .
>
> The second part is the fr.yaml file which should be an EXACT copy of the
> en.yaml with the things in quotes translated to French. In this PR you
> changed the formatting of the file, changed the order of some template
> strings, and changed some template strings. You also changed some of the
> variables too (Items in double curly braces). Take a note from the merges
> that were successfully incorporated and try to make this one match. Also
> please test and make sure that the fr.yaml actually works as I don't
> speak French and I wouldn't be able to really test it.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/stevenleeg/geemusic/pull/227#issuecomment-397051876>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/Aa1-1eOFgG882CKVM2JEMKon_pO4SMBnks5t8WNWgaJpZM4UmUBS>
> .
>
